src/toffy/fov_watcher.py

```diff
@@ -44,18 +52,16 @@ def __init__(self, run_folder: str, timeout: int = 10 * 60):
             if run_order * scan < 0:
                 raise KeyError(f"Could not locate keys in {run_folder}.json")
 
-            fov_names = [f"fov-{run_order}-scan-{s + 1}" for s in range(scan)]
-
-            # identify moly points
+            # scan 2's don't contain significant imaging data per new MIBI specs
+            fov_name = f"fov-{run_order}-scan-1"
```

Comment:
What information does a scan-2 contain?

Reply:
To my knowledge, `scan-2` is generally just Moly points anyways.